packages/neo-one-website/static.config.js

```diff
@@ -38,6 +40,16 @@ export default {
         path: '/404',
         component: 'src/pages/404',
       },
+      {
+        path: '/docs',
```

Comment:
What happens when you hit `/docs`? This should probably redirect to the first docs page (if it doesn't already).

Reply:
Make this go to 'src/pages/docs' with a `Redirect` component if the path is just `docs`